src/main/java/org/wso2/carbon/module/swiftiso20022/constants/MT940Constants.java

```diff
@@ -27,4 +27,84 @@ public class MT940Constants {
     public static final String TEXT = "text";
     public static final String CONTENT = "content";
     public static final String AXIS2_NS = "axis2ns";
+
+    //MT940 Related Constants
+    public static final String MT940_TRANSACTION_REF = ":20";
+    public static final String MT940_RELATED_REF = ":21";
+    public static final String MT940_ACCOUNT_NO = ":25";
+    public static final String MT940_STATEMENT_NO = ":28";
+    public static final String MT940_INFORMATION = ":86:";;
+    public static final String MT940_STATEMENT_LINE = ":61";
+    public static final String MT940_OPENING_BAL = ":60";
+    public static final String MT940_CLOSING_BAL = ":62";
+    public static final String MT940_CLOSING_AVAIL_BAL = ":64";
+    public static final String MT940_FORWARD_AVAIL_BAL = ":65";
+    public static final String SWIFT_TRANSFER = "S";
+    public static final String NON_SWIFT_TRANSFER = "N";
+    public static final String FIRST_ADVICE = "F";
+    public static final String CURRENT_STATEMENT_TYPE = "current";
+    public static final String LAST_STATEMENT_TYPE = "last";
+
+    //MT940 Constants display names
+    public static final String DN_HEADER_BLOCK_1 = "Header block 1";
+    public static final String DN_HEADER_BLOCK_2 = "Header block 2";
+    public static final String DN_HEADER_BLOCK_3 = "Header block 3";
+    public static final String DN_TRANSACTION_REF = "Transaction Reference Number";
+    public static final String DN_RELATED_REF = "Related Reference";
+    public static final String DN_SEQUENCE_NO = "Sequence Number";
+    public static final String DN_ACC_NUMBER = "Account Number";
+    public static final String DN_ACC_NUMBER_IDENTIFICATION = "Account Number Identification";
+    public static final String DN_ACC_IDENTIFICATION = "Account Identification";
+    public static final String DN_STATEMENT_NUMBER = "Statement Number";
+    public static final String DN_OPENING_BALANCE = "Opening Balance";
+    public static final String DN_TRANSACTION = "Transaction";
+    public static final String DN_AMOUNT = " Amount";
+    public static final String DN_CURRENCY = " Currency";
```

Comment:
Do these need to be start with a space prefix?
```suggestion
    public static final String DN_AMOUNT = "Amount";
    public static final String DN_CURRENCY = "Currency";
```

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/6/commits/8076e9b5033a9608b6839de38320ec994e01055d